app/javascript/components/MetaTags.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react"
+
+import Head from "next/head"
+
+export interface MetaTagsProps {
+  title: string
+  image?: string
+  description: string
+}
+
+const MetaTags = (props: MetaTagsProps) => (
```

Comment:
After bumping to 1.0.0 it looked like this Metatags component was removed from ui-components, does that sound right, @software-project?

I just copied this code directly from the public reference app.

Reply:
Thats correct. They were moved to the reference app.